pkg/flagbuilder/build_flags.go

```diff
@@ -131,6 +131,12 @@ func BuildFlags(options interface{}) (string, error) {
 				flag = fmt.Sprintf("--%s=%s", flagName, vString)
 			}
 
+		case *string:
```

Comment:
I think adding a unit test in the BuildFlags for this scenario is a good idea. Specially, if the 'if' condition was modified to fix a specific problem.